package.json

```diff
@@ -0,0 +1,44 @@
+{
+  "name": "react-tmp",
+  "version": "1.0.0",
+  "main": "index.js",
+  "license": "MIT",
+  "devDependencies": {
```

Comment:
I advise you to change the order of sections in `package.json` to specify the most significant parts (name, version, license, scripts) on top, and after the rest of files go. You can take a look at my example [here](https://github.com/dotnet-fizzyy/VueJS-course/blob/main/package.json#L1). The best way, in my opinion, is to structure in the following order I have shared link above (you and other people don't have to scroll file to see some commands to interact with etc).